testsuite/features/secondary/srv_monitoring.feature

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019 SUSE LLC
+# Copyright (c) 2019-2020 SUSE LLC
 # Licensed under the terms of the MIT license.
```

Comment:
Off-topic: 
I wouldn't assume that monitoring is enabled via sumaform, instead, I would start with a scenario enabling monitoring, and then another scenario disabling it. What do you think guys?

Reply:
The real problem is that this feature is not idempotent, and you will not solve that by splitting it. Also we have enough features like this, let's not add more.

Okay for a QA debt card, but it should be "make that feature idempotent", not "split that feature".

More important, that feature does not really test monitoring, it just plays with options. We need a real monitoring feature, where we really inspect monitoring data of client VMs. But that is another story.

Reply:
I didn't mention splitting the feature.
Just to reverse the order of the scenarios, first enable, then disable, and not let sumaform the work of the pre-condition, enabling the monitoring.
But yes, sure, this is a QA Debt card, nothing really important.

Reply:
ah, sorry, I misread.

If you change the order you might still have an idempotency problem, depending on current statuse of monitoring, even if we remove that in sumaform.

Imagine for example that test execution aborts in the middle of one of these series of scenarios.

Reply:
https://github.com/SUSE/spacewalk/issues/11967